source/posix/socket.c

```diff
@@ -674,7 +674,7 @@ int aws_socket_connect(
 
     if (pton_err != 1) {
```

Comment:
And for IOCP, it would be here:
https://github.com/awslabs/aws-c-io/blob/c4b661f44497b18201b56ffd200cc478441f6434/source/windows/iocp/socket.c#L692-L702

and here: https://github.com/awslabs/aws-c-io/blob/c4b661f44497b18201b56ffd200cc478441f6434/source/windows/iocp/socket.c#L753-L763